app/controllers/home_controller.rb

```diff
@@ -17,7 +17,7 @@ def diffusion_map
     @successful_ct = @diffusion_histories.get_by_successful_status.size
     @in_progress_ct = @diffusion_histories.get_by_in_progress_status.size
     @unsuccessful_ct = @diffusion_histories.get_by_unsuccessful_status.size
-
+    @include_google_maps = true
```

Comment:
🤔 Let's talk to Becca about the future designs for the homepage. I think we used to have the map on the homepage but removed it once the number of adoptions scaled. We might be able to remove this code from the home controller entirely!

Reply:
yes thanks for pointing this out, I added this to satisfy failing specs and meant to follow up on figuring out if we even wanted to keep the map capability on the home page